tools/windows-node-installer/pkg/cloudprovider/aws/ec2.go

```diff
@@ -580,7 +577,38 @@ func (a *AwsProvider) addPortToSg(sgId string, port int64) error {
 	return nil
 }
 
-// findWindowsWorkerSg creates the Windows worker security group with name <infraID>-windows-worker-sg.
+// createNewRulesForSg creates a new list of rules to be added in newly created sg
+func createNewRulesForSg(vpc *ec2.Vpc) ([]*ec2.IpPermission, error) {
+	myIP, err := GetMyIp()
```

Comment:
We should consider avoid calling this multiple times and get this information in a centralized place